pkg/cmd/roachtest/tests/django.go

```diff
@@ -27,8 +27,8 @@ import (
 var djangoReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<minor>\d+)(\.(?P<point>\d+))?$`)
 var djangoCockroachDBReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<minor>\d+)$`)
 
-var djangoSupportedTag = "cockroach-4.0.x"
-var djangoCockroachDBSupportedTag = "4.0.*"
+var djangoSupportedTag = "cockroach-4.1.x"
+var djangoCockroachDBSupportedTag = "4.1"
```

Comment:
This should be `4.1.*` (with dot star) to always use the latest minor release of django-cockroachdb.